script.sh

```diff
@@ -0,0 +1,32 @@
+!#/bin/bash
+home_path=/home/vvi/Ker_Modules/chtest/
```

Comment:
Please fix PATH.

No hardcoded paths! 

Make out the variable, set the default value, give the opportunity to change the variable